cluster/gce/util.sh

```diff
@@ -1595,7 +1597,7 @@ function create-certs {
   # Determine extra certificate names for master
   local octets=($(echo "${SERVICE_CLUSTER_IP_RANGE}" | sed -e 's|/.*||' -e 's/\./ /g'))
   ((octets[3]+=1))
-  local -r service_ip=$(echo "${octets[*]}" | sed 's/ /./g')
+  local -r service_ip=${octets[*]// /}
```

Comment:
I don't think this is doing the same thing, how does this replace spaces with periods?

Reply:
You are right, this is wrong. I looked into this creatuon of `service_ip` and found a much simpler way to make shellcheck happy about this with 2 lines of POSIX shell-internals. Coming up.